### PR TITLE
ansible-galaxy: add page

### LINKED ISSUE #1181: Add ansible-galaxy
A part of the Ansible toolset, it lets developers install third-party plugins (Ansible roles)

This is a separate CLI tool from `ansible`

### PR DESCRIPTION
**Pull request by brandonsoto**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines 

Fixes #1181

### CODE REVIEW

File: pages/common/ansible-galaxy.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# ansible-galaxy
+
+> Create and manage Ansible roles.
+
+- Install a role:
+
+`ansible-galaxy install {{role_name}}`
```
Comment (agnivade) [inline pages/common/ansible-galaxy.md]: Shouldn't it be username.role_name ?

File: pages/common/ansible-galaxy.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Remove a role:
+
+`ansible-galaxy remove {{role_name}}`
+
+- List installed roles:
+
+`ansible-galaxy list`
+
+- Search for roles:
```
Comment (agnivade) [inline pages/common/ansible-galaxy.md]: Change this to "Search for a given role"

**Comment by brandonsoto** [comment]
Thanks for the suggestions!

**Comment by sbrl** [comment]
Cool, thanks @brandonsoto :smiley_cat:

**Pull request merged**